client-app/yarn.lock

```diff
@@ -240,25 +240,25 @@
   dependencies:
     "@babel/highlight" "^7.12.13"
 
-"@babel/compat-data@^7.13.11", "@babel/compat-data@^7.13.15", "@babel/compat-data@^7.13.8", "@babel/compat-data@^7.14.0":
```

Comment:
This change to `yarn.lock` is the result of you running `yarn upgrade` locally, which is totally fine to do, but ideally not part of this diff (assuming that the upgraded deps are not important to the new functionality). 

I'd revert this file on the branch as a best practice.

If you did want to ensure that Toolbox was set to use the latest Hoist, you can run `yarn upgrade` on the develop branch, verify and push that, then do a catch-up merge to get the contacts branch even with develop.

Of course this would *not* apply if the PR included a deliberate change to dependencies that should get merged along with it.